content/metadata/2023/menu.yml

```diff
@@ -7,7 +7,7 @@
   - name: Call for Proposals
     url: program.html
   - name: Schedule
-    url: https://pretalx.com/euroscipy-2023/schedule/
+    url: 'schedule.html'
```

Comment:
Could you revert and keep the build-in pretalx version

Reply:
it's better to have a single domain for the event, at least at the discovery phase (when people might be searching for "EuroSciPy schedule"), we can change it once we'll have late-bird tickets